scripts/prep/aqm/exevs_aqm_prep.sh

```diff
@@ -160,28 +164,28 @@ fi
 
 if [ $hour -eq 12 ]
 then
-    ozmax8_file=$COMINaqm/${dirname}.${VDATE}/aqm.t${hour}z.max_8hr_o3${bctag}.${gridspec}.grib2
+    ozmax8_file=${COMINaqm}/${dirname}.${VDATE}/${hour}/aqm.t${hour}z.max_8hr_o3${bctag}.${gridspec}.grib2
     if [ -s ${ozmax8_file} ]; then
         wgrib2 -d 1 ${ozmax8_file} -set_ftime "0-23 hour ave fcst" -grib out1.grb2
         wgrib2 -d 2 ${ozmax8_file} -set_ftime "24-47 hour ave fcst" -grib out2.grb2
         wgrib2 -d 3 ${ozmax8_file} -set_ftime "48-71 hour ave fcst" -grib out3.grb2
-	if [ $SENDCOM = "YES" ]; then
-         cat out1.grb2 out2.grb2 out3.grb2 > ${COMOUT}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME}/aqm.t${hour}z.max_8hr_o3${bctag}.${gridspec}.grib2
+	if [ ${SENDCOM} = "YES" ]; then
+            cat out1.grb2 out2.grb2 out3.grb2 > ${COMOUTproc}/aqm.t${hour}z.max_8hr_o3${bctag}.${gridspec}.grib2
 	fi
     else
-      if [ $SENDMAIL = "YES" ]; then
-        export subject="t${hour}z OZMAX8${bctag} AQM Forecast Data Missing for EVS ${COMPONENT}"
-        echo "Warning: No AQM OZMAX8${bctag} forecast was available for ${VDATE} t${hour}z" > mailmsg
-        echo "Missing file is ${ozmax8_file}" >> mailmsg
-        echo "Job ID: $jobid" >> mailmsg
-        cat mailmsg | mail -s "$subject" $maillist
-      fi
+        if [ ${SENDMAIL} = "YES" ]; then
+            export subject="t${hour}z OZMAX8${bctag} AQM Forecast Data Missing for EVS ${COMPONENT}"
+            echo "Warning: No AQM OZMAX8${bctag} forecast was available for ${VDATE} t${hour}z" > mailmsg
```

Comment:
@Ho-ChunHuang-NOAA Can you please capitalize this WARNING as well? This capitalization is an NCO requirement.

Reply:
@AliciaBentley-NOAA  done